Changelog.md

```diff
@@ -2,24 +2,26 @@
 
 ## 0.0.22
 
+- Add compatibility with VS Code Quarkus
+
 ## 0.0.21
 
 - In Properties file, completion for possible enum values and booleans of a Camel component property 
-- In Properties file, the default values are automatically added when auto-completing Camel component properties
-- In Properties file, provide filtered completion when in middle of a component id, component property or value. Previously, completion was available only right after the dot or equal separators.
-- In Properties file, support insert-and-replace completion
+- In Properties file, the default values are automatically added when auto-completing Camel component properties

```

Comment:
I don't know why this line is changed. Until now, several unfructuous attempt to avoid the modification